source/includes/fundamentals/code-examples/Indexes.cs

```diff
@@ -0,0 +1,218 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using MongoDB.Driver.GeoJsonObjectModel;
+
+public class Indexes
+{
+
+    public static void Main(string[] args)
+    {   
+        // begin-declaration
+        // Replace with your connection string
+        const string uri = "<YOUR_CONNECTION_STRING>";
+
+        var mongoClient = new MongoClient(uri);
+        var database = mongoClient.GetDatabase("sample_mflix");
+        var movieCollection = database.GetCollection<Movie>("movies");
+        var theaterCollection = database.GetCollection<Theater>("theaters");
+        // end-declaration
+
+        SingleIndex(movieCollection);
+        CompoundIndex(movieCollection);
+        MultiKeyIndex(movieCollection);
+        TextIndex(movieCollection);
+        GeoSpatialIndex(movieCollection);
+        UniqueIndex(theaterCollection);
+        WildcardIndex(theaterCollection);
+    }
+
+    private static void SingleIndex(IMongoCollection<Movie> collection)
+    {
+        Console.WriteLine("single index");
+
+        // begin-single-index
+        collection.Indexes.CreateOne("{ title : 1 }");
```

Comment:
As noted above, I would steer users to the strongly typed `Builders<T>.IndexKeys` builder for defining indexes.